src/main/java/org/sonarlint/intellij/SonarLintIntelliJClient.kt

```diff
@@ -438,13 +440,13 @@ object SonarLintIntelliJClient : SonarLintRpcClientDelegate {
     }
 
     private fun setUpManualConnection(serverUrl: String): AssistCreatingConnectionResponse {
-        val warningTitle = "Trust This SonarQube Server?"
+        val warningTitle = "Trust This SonarQube Server Instance?"
         val message = """
-                        The server <b>${StringEscapeUtils.escapeHtml4(serverUrl)}</b> is attempting to set up a connection with SonarLint. Letting SonarLint connect to an untrusted SonarQube server is potentially dangerous.
+                        The server <b>${StringEscapeUtils.escapeHtml4(serverUrl)}</b> is attempting to set up a connection with SonarQube for IntelliJ. Letting SonarQube for IntelliJ connect to an untrusted SonarQube Server instance is potentially dangerous.
                         
                         If you don’t trust this server, we recommend canceling this action and <a href="$CONNECTED_MODE_SETUP_LINK">manually setting up Connected Mode<icon src="AllIcons.Ide.External_link_arrow" href="$CONNECTED_MODE_SETUP_LINK"></a>.
                     """.trimIndent()
-        val connectButtonText = "Connect to This SonarQube Server"
+        val connectButtonText = "Connect to This SonarQube Server Instance"
```

Comment:
Here it is uppercase again, please check all occurrences of `instance` and `organization`

Reply:
This is a requirement by the [UI guidelines](https://jetbrains.github.io/ui/text/capitalization/#title) from IntelliJ:
An example:
> Check Out from Version Control — out is a part of the phrasal verb check out and is capitalized.

Reply:
Alright!